src/parseTools.js

```diff
@@ -8,6 +8,58 @@
 
 //"use strict";
 
+var makePPEnum = (function() {
```

Comment:
this change could be a separate PR?

I think the topic was discussed before, and the hope was we can avoid adding a full parser like this, and can use eval instead?